lib/tasks/release.rake

```diff
@@ -0,0 +1,36 @@
+namespace :ndr_dev_support do
+  namespace :gem do
+    desc 'Quits if the current version has already been tagged.'
+    task :exit_if_already_released do
+      gemspec_path = Dir['{,*}.gemspec'].first
+      raise 'No gemspec found!' unless gemspec_path
+
+      gemspec = Bundler.load_gemspec(gemspec_path)
+      release_tag = "v#{gemspec.version}"
+
+      existing_tags = `git tag`
+      raise 'Error getting git tags!' if $?.exitstatus > 0
+
+      if existing_tags.split("\n").include?(release_tag)
+        abort "The tag for the release (#{release_tag}) already exists; stopping build process."
```

Comment:
I'm not sure if using `abort` here is ideal, as it would cause any CI job to fail. We may want to try and make this something that can cause a CI job to skip instead?

Reply:
Resolved in 788ae9b.